README.md

```diff
@@ -0,0 +1,24 @@
+# EPICS IOC for manage uTCA crates
+
+[![Continuous Integration Status](https://github.com/lnls-dig/ipmi-mgr-epics-ioc/actions/workflows/build.yml/badge.svg)](https://github.com/lnls-dig/ipmi-mgr-epics-ioc/actions)
+
+This repository contains the EPICS Input/Output Controller (IOC) used at LNLS
+based on [ipmiComm](https://github.com/slac-epics-modules/ipmiComm) support module.
+
+## Running the IOC
+
+You can use the following command to run it in the background using the default
+start-up script from
+[epics-in-docker](https://github.com/cnpem/epics-in-docker).
+
+```
+docker compose up -d
+```
+
+## Building the IOC image
+
+You can build the IOC with the following command:
+
+```bash
+TAG=latest docker compose build
```

Comment:
Since you don't use `TAG` in your docker-compose.yml, there is no need to define it here.